scripts/live-canary/test_scrub_artifacts.py

```
@@ -128,15 +238,210 @@ def test_strict_scrub_deletes_unsafe_artifacts_and_fails(self) -> None:
+            )
+
+            self.assertEqual(result.returncode, 1, result.stdout)
+            self.assertFalse(manifest.exists())
+
+    def test_strict_scrub_prunes_verified_nearai_runtime_manifest(self) -> None:
+        for endpoint in (
+            "https://cloud-api.near.ai/mcp",
+            "https://private.near.ai/mcp",
+        ):
+            with self.subTest(endpoint=endpoint):
+                with tempfile.TemporaryDirectory() as tmpdir:
```

Ruff (0.15.21)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)